web/packages/test/config/beacon-relay.json

```diff
@@ -18,7 +18,8 @@
   "sink": {
     "parachain": {
       "endpoint": "ws://127.0.0.1:11144",
-      "maxWatchedExtrinsics": 8
+      "maxWatchedExtrinsics": 8,
+      "headerRedundancy": 20
```

Comment:
Can we just move the new config `headerRedundancy` to `config.SpecSettings`, then not much change required.

Reply:
That's true, but it is not really related to the Ethereum spec settings, which is why I didn't add it there.